ui/src/pages/QuickSetup/FinishPage.tsx

```diff
@@ -60,29 +60,46 @@ const LoginTileGrid = () => {
   }
 
   return (
-    <div className="setup-page finish">
-      {identities.map(p => (
-        <div
-          className="log-in-tile"
-          key={p.payload.customer}
-          onClick={() => loginUser(dispatch, history, computeCredentials(p.payload.customer))}
-        >
-          <div className="log-in-row page-row">
-            <h4>{p.payload.legalName}</h4>
-            <p className="p2 log-in page-row">
-              Log in <ArrowRightIcon />
-            </p>
-          </div>
-          <p className="finished-roles">
-            {allRoles
-              .filter(r => r.contract.payload.provider === p.payload.customer)
-              .map(r => r.role)
-              .join(',')}
-          </p>
+    <>
+      <div className="setup-page finish">
+        <p className="dark details">cmd + click on a tile to Log In on a new tab </p>
+        <div className="log-in-tile-grid">
+          {identities.map(p => (
+            <div
+              className="log-in-tile"
+              key={p.payload.customer}
+              onClick={e => handleClick(e, p.payload.customer)}
+            >
```

Comment:
Not all users will be on mac... windows/linux uses "ctrl + click" for opening a link in a new tab. 

I think a possibly better option is
```
<Link to={location => {  
       loginUser(dispatch, history, computeCredentials(party), newWindow); 
       return '/app' ;
} >
``` 

I don't know for sure if this function will work like this (the react router docs show it being used to modify the existing location https://reactrouter.com/web/api/Link/to-function ) 

but if it does, then the ctrl + click / cmd + click should work naturally across all platforms/browsers. 

Then we can just style the Link components like the tiles

Reply:
I see, will give it a try!